parseclj-lex.el

```diff
@@ -229,25 +271,29 @@ behavior."
           (right-char)
           (parseclj-lex-token :lex-error (buffer-substring-no-properties pos (point)) pos :error-type :invalid-keyword))
       (progn
-        (while (or (parseclj-lex-symbol-rest? (char-after (point)))
+        (while (or (parseclj-lex-symbol-rest-p (char-after (point)))
                    (equal (char-after (point)) ?#))
           (right-char))
         (parseclj-lex-token :keyword (buffer-substring-no-properties pos (point)) pos)))))
 
 (defun parseclj-lex-comment ()
+  "Return a lex token representing a comment."
   (let ((pos (point)))
     (goto-char (line-end-position))
     (when (equal (char-after (point)) ?\n)
       (right-char))
     (parseclj-lex-token :comment (buffer-substring-no-properties pos (point)) pos)))
 
 (defun parseclj-lex-next ()
-  (if (parseclj-lex-at-eof?)
+  "Consume characters at point and return the next lexical token.
+
+See `parseclj-lex-token'."
```

Comment:
I'm not really happy with this docstring, I feel it should have a little bit more content, but I'm not sure what else to put here. thoughts?